tests/js/sentry-test/select.js

```diff
@@ -51,3 +51,14 @@ export function selectByValue(wrapper, value, options = {}) {
     .at(options.at || 0)
     .simulate('mouseDown');
 }
+
+export function selectByQuery(wrapper, query, options = {}) {
```

Comment:
@mgaeta Hmmm...we will have to update to using the `select-new.js` file once we stop using the deprecated select component. So this change might go away. IMO it might make more sense to make that change before adding tests but if you prefer doing the tests first that also can work.

Reply:
I've updated the component and switched to `select-new.js` here: https://github.com/getsentry/sentry/pull/23122